kore/src/Kore/Step/Simplification/Application.hs

```diff
@@ -104,11 +108,14 @@ evaluateApplicationFunction
         , SortedVariable variable
         , MonadSimplify simplifier
         )
-    => ExpandedApplication variable
+    => Predicate variable
+    -- ^ The predicate from the configuration
+    -> ExpandedApplication variable
     -- ^ The pattern to be evaluated
     -> simplifier (OrPattern variable)
-evaluateApplicationFunction Conditional { term, predicate, substitution } =
+evaluateApplicationFunction configurationPredicate Conditional { term, predicate, substitution } =
```

Comment:
The style guide recommends to keep lines under 80 chars (https://github.com/kframework/kore/wiki/Coding-Standards).